### ISSUE TITLE
no gives me a green check, cancel gives me yes

### ISSUE THREAD

**Issue by Primajin** [terkelg/prompts#180]
### Describe the bug
I have two unexpected behaviours regarding `confirm` prompts, where no gives me a green check and cancel gives me yes.

### To Reproduce

```js
const prompts = require('prompts');

(async () => {
    const response = await prompts({
        type: 'confirm',
        name: 'value',
        message: 'Do you want to upgrade: stuff?',
        initial: true
    });

    console.log(response.value);
})();
```

Given the above code:
- a `no` and a green check mark on <kbd>N</kbd>
- a `yes` and a red cross on <kbd>CTRL</kbd>+<kbd>C</kbd>
- a `yes` and a green check mark on <kbd>enter</kbd>
- a `yes` and a green check mark on <kbd>Y</kbd>

### Expected behavior
- a `no` and a red cross on <kbd>N</kbd>
- a `no` and a red cross on <kbd>CTRL</kbd>+<kbd>C</kbd>
- a `yes` and a green check mark on <kbd>enter</kbd>
- a `yes` and a green check mark on <kbd>Y</kbd>

### System

- OS: 10.14.5
- Terminal: iTerm
- Node version: v12.5.0

**Comment by lumio** [comment]
This kind of sounds like the expected behaviour to me.

If you had multiple prompts, each finished question gets a green check mark, once you've finished it. On the other hand if you cancel the prompt or an error occurred or you cancelled the prompt, a red cross is shown.

**Comment by Primajin** [comment]
Hmm OK... Feels weird. And why gives cancel yes? Because of the initial: true?

**Comment by lumio** [comment]
I agree, especially if you only have yes/no questions. What would be interesting to have might be a way to hide the red and green symbols, so it wouldn't be as confusing.

`yes` actually shouldn't be rendered here imo. When the questionnaire is being cancelled, nothing should be printed. So that looks like a bug to me.

**Comment by Primajin** [comment]
I think also an option to show an X as no would be good.

Imagine answering 10 yes/no questions:

✘  Question 1 … no
✔ Question 2 … yes
✔ Question 3 … yes
✔ Question 4 … yes
✔ Question 5 … yes
✔ Question 6 … yes
✔ Question 7 … yes
✘  Question 8 … no
✔ Question 9 … yes
✔ Question 10 … yes

You can immediately spot where you choose yes or no. With color for most people even easier.

**Comment by terkelg** [comment]
I think this is something that would be implemented in Prompts v3

**Comment by Primajin** [comment]
Hmm ok 🥺

**Comment by nurse-the-code** [comment]
Alternative suggestion: allow users to toggle on and off the checkmark. Because this also seems awkward:
<img width="452" alt="Screenshot 2024-12-25 at 22 45 45" src="https://github.com/user-attachments/assets/f43a01b6-55b7-4a3f-bc6c-8bdb56532213" />

in this larger context:
<img width="461" alt="Screenshot 2024-12-25 at 22 45 12" src="https://github.com/user-attachments/assets/8699e79e-57e6-4f3c-8e54-119007f3d36d" />
